suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -744,11 +753,14 @@ export const isAccountOutdated = (account: Account, freshInfo: AccountInfo) => {
                 // changed rewards amount (rewards are distributed every epoch (5 days))
                 freshInfo.misc!.staking?.rewards !== account.misc.staking.rewards ||
                 // changed stake pool
-                freshInfo.misc!.staking?.poolId !== account.misc.staking.poolId
+                freshInfo.misc!.staking?.poolId !== account.misc.staking.poolId ||
+                areDifferentTokens()
```

Comment:
total amount of txs changes when you receive tokens so no changes here are needed